hazelcast/src/test/java/com/hazelcast/concurrent/lock/LockAdvancedTest.java

```diff
@@ -388,7 +388,7 @@ public void run() {
         t.start();
         sleepMillis(5000);
         t.interrupt();
-        assertTrue(latch.await(15, TimeUnit.SECONDS));
+        assertOpenEventually(latch, 30);
```

Comment:
this is the only change - timeout is increased. everything else is just a conversion of assertions.